src/docs/sphinx/pythonTools/mesh_doctor.rst

```diff
@@ -122,3 +122,60 @@ It will also verify that the ``VTK_POLYHEDRON`` cells can effectively get conver
 
 .. command-output:: python mesh_doctor.py supported_elements --help
    :cwd: ../../../coreComponents/python/modules/geosx_mesh_doctor
+
+``Using mesh_doctor in paraview``
+""""""""""""""""""""""""""""""""""
+To use ``mesh_doctor`` in Paraview as a python programmable filter, a python package install is required first in Paraview python resolved
+path. Paraview is storing its python ressources under its *lib/pythonX.X* depending on the paraview version, *e.g* Paraview 5.11 is working
+with python 3.9. As a results the following command will install ``mesh_doctor`` package into Paraview resolved path.
+
+.. command-output:: python3 -m pip install --index-url https://test.pypi.org/simple/ --no-deps --upgrade --target /path/to/Paraview/lib/python3.9/ mesh_doctor
+
+.. note::
+    ``pip`` is installing the ``mesh_doctor`` package from the test.pypi repo, which is intended to test package deployment.
+    Once stabilized and ``mesh_doctor`` uploaded onto the main package repo, this should be dropped out.
+
+Once the installation done, the */path/to/Paraview/lib/pythonX.X* should holds ``mesh_doctor`` package content, *i.e.* ``checks`` and ``parsing``.
+Then launching ``Paraview`` and loading our *mesh.vtu*, as an example, we will design a *Programmable python filter* relying on *element_volumes* from
+``mesh_doctor``. Add such a filter pipelined after the mesh reader, in the script section paste the following,
+
+.. code-block:: python
+    :linenos:
+
+    input0 = inputs[0]
+    tol = 1.2e-6
+
+    from checks import element_volumes
+    import vtk
+    #
+    res = element_volumes.__check(inputs[0].VTKObject, element_volumes.Options(tol))
```

Comment:
We'll need to change the name of `__check` 🤣 
Also, if using `inputs[0]` is an issue, maybe `mesh_doctor` can evolve?

Reply:
Agreed, going into _private_ function is hacky :stuck_out_tongue:

Reply:
Yes, that will definitely be interesting for GEOS user working with external meshes.
@jafranc Have you considered Paraview Python plugin instead of Programmable Filter in Paraview?

Reply:
Perhaps @AntoineMazuyer might be able to help on this